packaging/win/delugewin.spec

```diff
@@ -0,0 +1,324 @@
+
+# -*- mode: python ; coding: utf-8 -*-
+from PyInstaller.compat import is_win
+from PyInstaller.utils.hooks import collect_all, copy_metadata
+
+datas = []
+binaries = []
+if is_win:
+    if "GITHUB_ACTIONS" in os.environ:
```

Comment:
This seems very fragile having github actions specific lines in the spec file. Also this is added for x86 arch which probably not what we want?

What is causing pyinstaller not to include them? Could this issue not always be the case any x64 build not just github actions?

Is there a way of specifying it via argument or perhaps without full path? e.g. `libcrypto-1_1-x64.dll`